u8timeseries/timeseries.py

```diff
@@ -678,92 +621,59 @@ def append_values(self,
             raise_if_not(index[-1] == index[0] + self.freq(),
                          'Appended index must have the same frequency as the current one.', logger)
         values = values[new_indices]
-        new_series = pd.Series(values, index=index)
+        new_series = pd.DataFrame(values, index=index)
         series = self._series.append(new_series)
-        if conf_lo is not None and self._confidence_lo is not None:
-            raise_if_not(len(index) == len(conf_lo), 'Confidence intervals must have same length as index.', logger)
-            conf_lo = conf_lo[new_indices]
-            conf_lo = self._confidence_lo.append(pd.Series(conf_lo, index=index))
-        if conf_hi is not None and self._confidence_hi is not None:
-            raise_if_not(len(index) == len(conf_hi), 'Confidence intervals must have same length as index.', logger)
-            conf_hi = conf_hi[new_indices]
-            conf_hi = self._confidence_hi.append(pd.Series(conf_hi, index=index))
 
-        return TimeSeries(series, conf_lo, conf_hi)
+        return TimeSeries(series)
 
     def update(self,
                index: pd.DatetimeIndex,
-               values: np.ndarray = None,
-               conf_lo: np.ndarray = None,
-               conf_hi: np.ndarray = None,
-               inplace: bool = False) -> 'TimeSeries':
+               values: np.ndarray = None) -> 'TimeSeries':
         """
-        Updates the Series with the new values provided.
+        Updates the TimeSeries instance with the new values provided.
         If indices are not in original TimeSeries, they will be discarded.
-        At least one parameter other than index must be filled.
         Use `numpy.nan` to ignore a specific index in a series.
 
-        It will raise an error if try to update a missing CI series
-
         Parameters
         ----------
         index
             A `pandas.DateTimeIndex` containing the indices to replace.
         values
             An array containing the values to replace (optional).
-        conf_lo
-            The lower confidence interval values to change (optional).
-        conf_hi
-            The upper confidence interval values (optional).
-        inplace
-            If True, do operation inplace and return self, defaults to False.
 
         Returns
         -------
         TimeSeries
-            A new TimeSeries (if `inplace = False`) or the same TimeSeries with values updated
+            A new TimeSeries with updated values.
         """
 
-        raise_if_not(not (values is None and conf_lo is None and conf_hi is None),
-                     "At least one parameter must be filled other than index", logger)
+        if isinstance(values, (list, range)):
+            values = np.array(values)
+
+        if values is not None and len(values.shape) == 1:
+            values = values.reshape((len(values), 1))
+
+        raise_if_not(not (values is None), "'values' parameter should not be None.", logger)
```

Comment:
maybe time for raise_if that would be basically implemented as not (raise_if_not)

Reply:
done